F1/F1/Standings/StandigsViewModel.swift

```diff
@@ -35,4 +36,41 @@ class StandingsViewModel {
             }
         }
     }
+
+    func isLoaded() -> Bool {
+        driverViewModel.isLoaded && constructorViewModel.isLoaded
+    }
+
+    func fetchStandings() {
+        let dispatchGroup = DispatchGroup()
```

Comment:
why do you need to use concurrency. If it is for loading, there are better ways to handle that with delegates and UIloaders

Reply:
chatted offline